service/matching/taskQueueManager.go

```diff
@@ -146,6 +146,12 @@ func withIDBlockAllocator(ibl idBlockAllocator) taskQueueManagerOpt {
 	}
 }
 
+func withFatalSignaler(f func(id *taskQueueID)) taskQueueManagerOpt {
```

Comment:
method not used

Reply:
Dang. Used in the next PR. I'll rebase it out.